riff-raff.yaml

```diff
@@ -22,7 +22,7 @@ deployments:
         CODE: ManageFrontend-CODE.template.json
         PROD: ManageFrontend-PROD.template.json
       amiTags:
-        Recipe: jammy-membership-node18
+        Recipe: jammy-membership-node22
```

Comment:
will we need to stop the node 18 one baking now (to save money), or is that still used by SDC?